manticore/seth.py

```diff
@@ -441,12 +631,9 @@ def load(self, state_id=None):
             if len(self.running_state_ids) == 1:  
                 #Get the ID of the single running state              
                 state_id = self.running_state_ids[0]
-                if state_id != -1:
-                    #if there is a single running state with id != 1. We consider it is a new initial_state
-                    assert self.initial_state is None
-                    state = self.initial_state = self._executor._workspace.load_state(state_id, delete=True)
             else:
-                raise Exception("More than one state running. Do not know which to choose.")
+                raise Exception("More than one state running. Do not know which one to choose.")
```

Comment:
for now, does it matter which state to choose? just choose the first?

Reply:
None, means "Give me the one state". It should fail if multiple states are enqueuedueue. 
I'll change the txt. to  "More than one state running."